### PR TITLE
Music

### PR DESCRIPTION
**Pull request by Saatvik-droid**
Added all music commands (play, pause, resume, stop, current).

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -6,50 +6,47 @@
...
+Loads environment variables from .env.
+Initializes TOKEN as bot token.
 """
 load_dotenv()
-DISCORD_TOKEN = os.environ.get('DISCORD_TOKEN')
-
-bot = commands.Bot(command_prefix='.')
+TOKEN = os.environ.get('TOKEN')
 
+bot = commands.Bot(command_prefix=',')
```
Comment (sayand0122) [inline bot.py]: change the prefix to '.'

**Review by sayand0122 — CHANGES_REQUESTED** [review]
Do the required changes and we are good to merge it

File: bot.py [hunk 2]
```diff
@@ -6,50 +6,47 @@
...
-    Loads environment variables from .env.
-    Initializes TOKEN as bot token.
+Loads environment variables from .env.
+Initializes TOKEN as bot token.
 """
 load_dotenv()
-DISCORD_TOKEN = os.environ.get('DISCORD_TOKEN')
-
-bot = commands.Bot(command_prefix='.')
+TOKEN = os.environ.get('TOKEN')
```
Comment (sayand0122) [inline bot.py]: just for clarity change TOKEN to DISCORD_TOKEN

File: bot.py (hunk 1 above)
Comment (Saatvik-droid) [reply bot.py]: Done. Take a look now.

**Pull request merged**